app/src/main/AndroidManifest.xml

```diff
@@ -144,15 +144,10 @@
                 android:resource="@xml/entity_widget_info" />
         </receiver>
 
-        <receiver android:name=".widgets.todo.TodoWidget" android:label="@string/widget_todo_label"
+        <receiver android:name=".widgets.todo.TodoWidgetReceiver" android:label="@string/widget_todo_label"
```

Comment:
this may be a breaking change for users, meaning if they had the widget before the update, it will be removed from the home screen after. I think the widgets are linked to the manifest name here, we had an old bug. An easy test is to deploy main and add the widget then update to this branch and see if the widget stays on the home screen. Last time we updated the widget file name we found out the hard way you lose widgets 😬

Reply:
I will try 👍

Reply:
Indeed it failed. I renamed the classes. But I would like your opinion to both of you with @jpelgrom. I named the Compose class `TodoGlanceAppWidget` and the receiver `TodoWidget`.

How should we name this classes for future widget. I would like to add it to the documentation.

Reply:
> `TodoWidget` for Compose and `TodoWidgetReceiver` for the receiver.

For consistency and to avoid breaking changes for other widgets that would be my preference (and the names are still logical to me)

Reply:
agreed, that makes sense to me as well